lib/src/non_fungible_token/io.rs

```diff
@@ -1,22 +1,22 @@
 use crate::non_fungible_token::{royalties::*, token::*};
 use gstd::{prelude::*, ActorId};
 
-#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq)]
+#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq, Clone, Copy)]
 pub struct NFTTransfer {
     pub from: ActorId,
     pub to: ActorId,
     pub token_id: TokenId,
 }
 
-#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq)]
+#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq, Clone)]
 pub struct NFTTransferPayout {
     pub from: ActorId,
     pub to: ActorId,
     pub token_id: TokenId,
     pub payouts: Payout,
 }
 
-#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq)]
+#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq, Clone, Copy)]
```

Comment:
Again, I mind deriving `Copy` for large structures.
```suggestion
#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq, Clone)]
```